pom.xml

```diff
@@ -3,7 +3,7 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-<!--    <packaging>jar</packaging>-->
+    <packaging>jar</packaging>
```

Comment:
the parent pom should use packaging `pom`.  there's nothing to package a jar for parent.